### DISCUSSION TITLE
How to display typescript errors in the console in development mode?

### DISCUSSION THREAD

**Discussion by odeyal0** [vercel/turborepo#1330] [Help] [answered]
Errors not displaying

![Screenshot from 2022-06-03 14-15-01](https://user-images.githubusercontent.com/46008279/171843633-ca7f5a3e-9152-4e53-9814-a77f6d43056f.png)

**Comment by anthonyshew** [comment, marked as answer]
The errors that show in your editor can get out of sync for several different reasons. Compiled targets aren't necessarily what VSCode sees. Not a Turborepo issue, thankfully!
